Apps/Playground/Scripts/experience.js

```diff
@@ -90,6 +90,111 @@ CreateBoxAsync().then(function () {
         scene.createDefaultLight(true);
     }
 
+    engine.updateDynamicTexture = function (texture, canvas, invertY, premulAlpha, format) {
```

Comment:
what are we doing so that folks don't have to add this javascript code for canvasess to function correctly?

Reply:
This code will end up being in nativeEngine.ts. I didn't open a PR for that yet because it may change a little during develpment and because I want the canvasAbstraction PR to be merged first.